ui/components/component-library/avatar-token/avatar-token.js

```diff
@@ -102,17 +102,17 @@ AvatarToken.propTypes = {
    * The background color of the AvatarToken
```

Comment:
This follows our other conventions of these propTypes. No propType errors in console.